d4287e1060c8892901baa10333ed29131d4fcdbe5d15c476994biff --git a/package/rpi-firmware/rpi-firmware.hash b/package/rpi-firmware/rpi-firmware.hash
index 833a7fbe37..01b2181d29 100644
--- a/package/rpi-firmware/rpi-firmware.hash
+++ b/package/rpi-firmware/rpi-firmware.hash
@@ -1,2 +1,2 @@
 # Locally computed
-sha256 0451e1a843bf02df81126a1625d9a15dfdf19dcb2f9e95b0613a63e59fd31ca8 rpi-firmware-2067241bc7cdf238fdeab7f5a3c22faa57dd5617.tar.gz
+sha256 e636895b34f5a4287e1060c8892901baa10333ed29131d4fcdbe5d15c476994b rpi-firmware-6e3c5f8d5fd9b070feb5b846abca3b52b0230439.tar.gz
